docs/content/en/docs/crd-ref/_index.md

```diff
@@ -13,3 +13,21 @@ that are defined for the Keptn Lifecycle Toolkit.
 Information that is published here has been reviewed for technical accuracy
 but the format and content is still evolving.
 We welcome your input!**
+
+Each CRD is an object of an API library.
+Keptn APIs follow the Kubernetes API versioning scheme.
+and is itself composed of objects and sub-objects.
+
+For more information, see the Kubernetes documentation:
+
+* [API Overview](https://kubernetes.io/docs/reference/using-api/)
+
+* [Custom Resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#:~:text=A%20resource%20is%20an%20endpoint,in%20a%20default%20Kubernetes%20installation.)
+
+* [API versioning](https://kubernetes.io/docs/reference/using-api/#api-versioning)
+
+* [Understanding Kubernetes Objects](https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/)
+
+
+
+
```

Comment:
Please, delete these lines to make our markdown linter happy :)

Reply:
Hrump!  I fixed the blank lines, which now adds crd-ref/_index.md to this PR.  The only changes to that file is deleting blank lines at the end.

I checked my command history and I used commit -s here so not sure what happened.  Following the instructions in github to rebase and all didn't work either.